### PR TITLE
Adding facetorch

### PR DESCRIPTION
**Pull request by tomas-gajarsky**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tomas-gajarsky** [comment]
I am willing to be listed in the package summary.

**Comment by tomas-gajarsky** [comment]
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/facetorch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+build:
+  skip: True  # [py<39 or win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
```
[the remark this answers is not in the document]
Comment (tomas-gajarsky) [reply recipes/facetorch/meta.yaml]: You are right, I removed the compiler

File: recipes/facetorch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{% set name = "facetorch" %}
+{% set version = "0.0.5" %}
```
[the remark this answers is not in the document]
Comment (tomas-gajarsky) [reply recipes/facetorch/meta.yaml]: I updated it to 0.0.6
Thank you for the review and suggestions! Let me know if there is anything else that I should do

**Comment by hmaarrfk** [comment]
thanks

**Pull request merged**